### PR TITLE
Added documentation for voltmeter

### LINKED ISSUE #20: No help on voltmeter
nest.help('voltmeter') 

yields

NESTError: NoHelpInformationError in help

### PR DESCRIPTION
**Pull request by heplesser**
Added documentation for voltmeter and removed or updated some *meter-… related comments. This should fix #20.

### CODE REVIEW

File: models/modelsmodule.cpp [hunk 1]
```diff
@@ -197,6 +197,71 @@ ModelsModule::init( SLIInterpreter* )
   register_model< volume_transmitter >( net_, "volume_transmitter" );
 
   // Create voltmeter as a multimeter pre-configured to record V_m.
+  /*BeginDocumentation
+  Name: voltmeter - Device to membrane potential from neurons.
```
Comment (jougs) [inline models/modelsmodule.cpp]: "record" missing

File: models/modelsmodule.cpp [hunk 2]
```diff
@@ -197,6 +197,71 @@ ModelsModule::init( SLIInterpreter* )
...
+
+  Remarks:
+   - The voltmeter model is implemented as a multimeter preconfigured to
+     record /V_m.
+   - The set of variables to record and the recording interval must be set
+     BEFORE the voltmeter is connected to any node, and cannot be changed
+     afterwards.
+   - A voltmeter cannot be frozen.
+   - If you record with voltmeter in accumulator mode and some of the nodes
+     you record from and others are not, data will only be collected from the
```
Comment (jougs) [inline models/modelsmodule.cpp]: "are frozen" missing between "from" and "and"

**Comment by jougs** [comment]
Thanks for taking care. I'm :+1: for merging after the issues above are fixed.

**Comment by tammoippen** [comment]
Nice. I did not know, model documentation could also go at this place. :+1: from my side.

**Comment by jougs** [comment]
@tammoippen: Actually, it can go wherever it wants. The help parser just scans all `.sli`, `.cc`, `.h`, and `.cpp` files and extracts the documentation. It only might put things into the `sli` help folder instead of the `cc` one, or vice versa. Those distinctions might go away anyway, though.

See [`helpinit.sli`](nest/nest-simulator/blob/master/lib/sli/helpinit.sli#L115) for details.

**Pull request merged**